Composer/packages/client/src/components/CreationFlow/v2/AddBotModal.tsx

```diff
@@ -16,17 +16,17 @@ interface AddBotModalProps {
   onDismiss: () => void;
 }
 
-const addExistingKey = 'existing';
-const addBlankKey = 'blank';
+const addExistingBotKey = 'existing';
+const addNewBotKey = 'new';
 
 const addSkillOptions: IChoiceGroupOption[] = [
-  { key: addExistingKey, text: 'Add an existing bot' },
-  { key: addBlankKey, text: 'Create a new blank bot' },
+  { key: addNewBotKey, text: 'Create a new bot' },
```

Comment:
these were missing formatMessage originally, can you add them?

Reply:
handled and made it a function